miniwin/miniwin/src/miniwin.cpp

```diff
@@ -0,0 +1,64 @@
+#include "miniwin.h"
+
+#include <SDL3/SDL.h>
+#include <utility>
+#include <vector>
+
+static std::vector<std::pair<SDL_Window *, HWND>> sdl_hwnd_mapping;
+
+SDL_Window *miniwin_GetSdlWindow(HWND hWnd)
+{
+    for (size_t i = 0; i < sdl_hwnd_mapping.size(); i++) {
+        if (sdl_hwnd_mapping[i].second == hWnd) {
+            return sdl_hwnd_mapping[i].first;
+        }
+    }
+    return NULL;
+}
+
+void OutputDebugString(const char* lpOutputString)
+{
+	SDL_LogDebug(SDL_LOG_CATEGORY_APPLICATION, "%s", lpOutputString);
+}
+
+VOID WINAPI Sleep(DWORD dwMilliseconds)
+{
+	SDL_Delay(dwMilliseconds);
+}
+
+BOOL GetWindowRect(HWND hWnd, tagRECT* Rect)
+{
+	int x, y, w, h;
+    if (!Rect) {
```

Comment:
Indentation is all over the place in a lot of the files

Reply:
Looks like fixing that also helped the diff :+1:

Reply:
I copied some stuff around, so my IDE was surely able to mess with everything. :)